Create arguments.md

kg6gfq opened the pull request:
with some preliminary arguments for why these changes are necessary.

kg6gfq committed:
Create arguments.md 

with some preliminary arguments for why these changes are necessary.

rynyday reviewed (approved):
Looks good! It's a great rough draft and start.

kg6gfq committed:
Added description at top

kg6gfq committed:
notes on FCC and doxxing

the FCC doesn't seem to take a position on doxxing generally, but here are a couple thoughts

rynyday reviewed (approved):
I think the "doxxing" of Ajit Pai might be useful if we can get the current FCC administrations opinions about it. But good point and it should be looked into. 

I think the FCC's mission to provide connectivity to DV victims is relevant and will definitely help our case. 

Citation for accessibility for legal name change is cake. Good job as always Darin, approving!
